### ISSUE TITLE
Installation misbehavior caused apache2 service failer

### ISSUE THREAD

**Issue by abenbachir** [microsoft/OMS-Agent-for-Linux#863]
After omsagent installation apache2 service no longer starts, because the following line was added to the apache2 config file "/etc/apache2/apache2.conf", but apache-cimprov was not installed :

`include /etc/opt/microsoft/apache-cimprov/conf/mod_cimprov.conf`

The source of this problem seem to be in "FindApacheConfigFile" function from file "opsmgr-kits/apache-oss-test.sh", for ubuntu type installations, the logic looks like this:
```
# Ubuntu-type installation
APACHE_CONF=/etc/apache2/apache2.conf 
CONF_STYLE=f 
[ -e "${APACHE_CONF}" ] && return
```
Workaroud must be to create a symlink of "/etc/opt/microsoft/apache-cimprov/conf/mod_cimprov.conf" under mods-enabled/, conf-enabled/ or sites-enabled/.

**Comment by saimsh** [comment]
@abenbachir  any update on this? just checking back  :)

**Comment by abenbachir** [comment]
@simathih @hestolz should be able to handle this.

**Comment by saimsh** [comment]
just some insights, 
I was working on a mediawiki installation and configuration, here, instead of directly feeding to the apache2.conf, 
i created a mediawiki.conf file under /etc/apache2/sites-available/ and added custom Virtual Host  entry for media wiki there.
Next, I simply enabled the the media wiki site and rewrote the module required in the following manner:
sudo a2ensite mediawiki.conf
sudo a2enmod rewrite

I am assuming that the above approach could also be used for other resources utilizing apache ( in this case  mod_cimprov.conf). If we are able to implement this, and in case of future issues with omi or the conf itself, the whole apache2 server will not be brought down, ensuring minimal impact to the webserver itself.

**Comment by abenbachir** [comment]
The following steps should be followed as a workaround:

```
$> ln -s /etc/opt/microsoft/apache-cimprov/conf/mod_cimprov.load /etc/apache2/mods-available/mod_cimprov.load
$> a2enmod mod_cimprov
$> systemctl restart apache2

check that module was loaded correctly 
$> apache2ctl -M | grep cim
	cimprov_module (shared) 
```

**Comment by saimsh-msft** [comment]
hi @abenbachir. 
Thanks. i have implemented the workaround for my customer.
I was wondering if this correct method could be added in a similar manner during omsagent installation itself.
thanks,
saim

**Comment by saimsh-msft** [comment]
Checking back for update on this as i believe the module should be added separately and not to the global config itself

**Comment by abenbachir** [comment]
I think we have added a fix for this issue in 1.12 release
